internal/system/agent/init.go

```diff
@@ -80,8 +80,6 @@ func newExecutorClient(operationsType string) (interfaces.ExecutorClient, error)
 	switch operationsType {
 	case "os":
 		return &executor.ExecuteOs{}, nil
-	case "docker":
```

Comment:
This is correct for now, but will change (later) _after_ another feature branch, per [PR#926](https://github.com/edgexfoundry/edgex-go/pull/926), has been merged into `master`.